models/reviews.models.js

```diff
@@ -45,30 +46,66 @@ exports.fetchCommentsByReviewId = (review_id) => {
     });
 };
 
-exports.fetchReviews = (category) => {
+exports.fetchReviews = ({ category, sort_by, order }) => {
   const validCategoryValues = [
     "euro_game",
     "social_deduction",
     "dexterity",
     "childrens_games",
+    undefined,
```

Comment:
This wouldn't be a valid value, there is a better way we could handle this x 3(in relation to your other green lists)